### PR TITLE
Bump the baseline

### PR DESCRIPTION
**Pull request by m-kuhn**

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for dfda06cd76beab00f5b02007845d38dfe57b712c: [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3350-802bad74d827801b94b96b3d53638c5c9bc93870-arm64-android.apk)
<details>
<summary>Other architectures</summary>

- [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3350-802bad74d827801b94b96b3d53638c5c9bc93870-arm-android.apk)
- [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3350-802bad74d827801b94b96b3d53638c5c9bc93870-x64-android.apk)
- [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3350-802bad74d827801b94b96b3d53638c5c9bc93870-x86-android.apk)
</details>

File: vcpkg/triplets/arm-android.cmake [hunk 1]
```diff
@@ -3,7 +3,7 @@ set(VCPKG_CRT_LINKAGE dynamic)
 set(VCPKG_LIBRARY_LINKAGE static)
 set(VCPKG_CMAKE_SYSTEM_NAME Android)
 set(VCPKG_BUILD_TYPE release)
+set(VCPKG_MAKE_BUILD_TRIPLET "--host=armv7a-linux-androideabi")
 
-
-set(VCPKG_CXX_FLAGS "-fstack-protector-strong -lunwind -Wl,--exclude-libs=libunwind.a")
-set(VCPKG_C_FLAGS "-fstack-protector-strong -lunwind -Wl,--exclude-libs=libunwind.a")
+set(VCPKG_CXX_FLAGS "-fstack-protector-strong")
```
Comment (nirvn) [inline vcpkg/triplets/arm-android.cmake:8]: The unwind stuff needs to be re-added here.

**Comment by nirvn** [comment]
The arm64 APK runs fine here.

File: vcpkg/triplets/arm-neon-android.cmake [hunk 2]
```diff
@@ -1,6 +1,9 @@
 set(VCPKG_TARGET_ARCHITECTURE arm)
-set(VCPKG_CRT_LINKAGE static)
+set(VCPKG_CRT_LINKAGE dynamic)
```
Comment (nirvn) [inline vcpkg/triplets/arm-neon-android.cmake:2]: I think you could kill this neon triplet file altogether btw

Comment (m-kuhn) [reply vcpkg/triplets/arm-neon-android.cmake:2]: Not at all, r25 drops non neon, we should drop the current arm apks instead. Neon is faster and supported by virtually any device.

**Comment by m-kuhn** [comment]
> The arm64 APK runs fine here.

Did you check if proj.db is fine? Extrapolating from the test failures that a path change has happened :thinking:

**Comment by nirvn** [comment]
@m-kuhn , we had added proj.db check in the cmakefiles.txt to avoid silently regressing, wondering why it failed us.

**Comment by nirvn** [comment]
@m-kuhn , it seems the safeguard was removed. IMHO worth adding something like this:

https://github.com/opengisch/QField/commit/e6fc6159424290d95b1cbffea68e94cbd74500f2

**Comment by m-kuhn** [comment]
Done.
What we should actually have: spix test running in an emulator

**Comment by nirvn** [comment]
@m-kuhn , yeah, that'd be good. The nice thing about the cmakelists check though is that it allows for the process to bail out earlier and reduces the amount of time waiting for that feedback.

**Comment by m-kuhn** [comment]
I hope the file isn't moved too often by upstream in the future (which as far as I can see is the only thing this check ensures) ...

**Comment by nirvn** [comment]
@m-kuhn , it moved twice already (once from OSGeo4A -> vcpkg, and once during a vcpkg update).

**Comment by m-kuhn** [comment]
> @m-kuhn , it moved twice already (once from OSGeo4A -> vcpkg, and once during a vcpkg update).

```
¯\_(ツ)_/¯
```

**Pull request closed without merging**